### DISCUSSION TITLE
I have a web3modal application that operates normally in the Safari browser on my phone. But when I put this web application in the webview of the app, I can open the metamask app. However, they are not communicating. No connection pop up. May I ask how to handle it?

### DISCUSSION THREAD

**Discussion by hinson888** [WalletConnect/walletconnect-monorepo#4533] [Developer Support]
I have a web3modal application that operates normally in the Safari browser on my phone. But when I put this web application in the webview of the app, I can open the metamask app. However, they are not communicating. No connection pop up. May I ask how to handle it?

**Comment by glitch-txs** [comment]
which app? could you share the steps to reproduce the issue?
